pkgs/mime/tool/mime_types.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:mime/mime.dart';
+import 'package:mime/src/default_extension_map.dart';
+
+void main(List<String> args) {
+  // Get all the mime types.
+  final mimeTypes = defaultExtensionMap.values.toSet().toList()..sort();
+
+  // Find all their extensions.
+  final mimeToExts = <String, List<String>>{};
+
+  for (final entry in defaultExtensionMap.entries) {
+    final ext = entry.key;
+    final mime = entry.value;
+
+    mimeToExts.putIfAbsent(mime, () => []).add(ext);
+  }
+
+  // Emit the table.
+  const marker = '<!-- table -->\n';
+  final file = File('doc/mime_types.md');
+  var contents = file.readAsStringSync();
+  final prefix =
+      contents.substring(0, contents.indexOf(marker) + marker.length);
+  final suffix = contents.substring(contents.lastIndexOf(marker));
+
+  final buf = StringBuffer();
```

Comment:
Put the prefix in to the buffer from the start.
```dart
  final buffer = StringBuffer(prefix);
```
and add the suffix at the end. Might as well use the buffer when we have it.